### PR TITLE
feat: automatic crawler name

### PR DESCRIPTION
**Pull request by damcou**
Set default crawler name related to repository and branch

### CODE REVIEW

**Comment by damcou** [comment]
Tested on https://github.com/damcou/simple-page/pull/5

**Review by bodinsamuel — APPROVED** [review]
LGTM

File: build/index.js [hunk 1]
```diff
@@ -402,7 +402,8 @@ var CRAWLER_USER_ID = core.getInput('crawler-user-id');
 var CRAWLER_API_KEY = core.getInput('crawler-api-key');
 var CRAWLER_API_BASE_URL = core.getInput('crawler-api-base-url');
 var GITHUB_TOKEN = core.getInput('github-token');
-var CRAWLER_NAME = core.getInput('crawler-name').replace(/\//g, '-');
+var CRAWLER_NAME = core.getInput('crawler-name').replace(/[ /]/g, '-');
```
Comment (bodinsamuel) [inline build/index.js:405]: we don't need to remove space neither slash for crawler name, it currently makes the name hardly readable.
the indexname spec should be covered by next replace

**Review by bodinsamuel — COMMENTED** [review]
Minor comment; also the title of the PR should be updated ☺️

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**